artipie-main/src/test/java/com/artipie/debian/DebianS3ITCase.java

```diff
@@ -97,6 +97,48 @@ void setUp() throws IOException {
         );
     }
 
+    @ParameterizedTest
+    @CsvSource({
+        "8080,my-debian,9000"
+    })
+    void curlPutWorks(final String port, final String repo, final String s3port) throws Exception {
```

Comment:
Why is this test `@ParameterizedTest`?

Reply:
@dgarus , it makes code in tests more concise and + allows to test several variants, in perspective.

Reply:
As I can see, this test is executed with just one `csv` line, so you can define `port`, `repo`, `s3port` as local variables. Isn't it?

Reply:
@dgarus , ok.